main.py

```diff
@@ -6,19 +6,22 @@
 from ChangeRephraser import ChangeRephraser
 from GithubPR import GithubPR
 from ReviewBuddy import ReviewBuddy
-
+from SecurityReviewBuddy import SecurityReviewBuddy
 PR_NUMBER = 5
 REPO_NAME = "mohmiim/Algo"
 TOKEN = open("token", "r").read()
 gitPR = GithubPR(REPO_NAME, PR_NUMBER, TOKEN)
 change = gitPR.get_all_changes_as_text()
 rephraser = ChangeRephraser()
 reviewer = ReviewBuddy()
+security_reviewer = SecurityReviewBuddy()
 description = rephraser.convert(change)
 gitPR.update_pr_description(description)
 for file, change in gitPR.get_change_for_review_as_text().items():
     print("Reviewing file: ", file)
     comment = reviewer.convert(change)
-    gitPR.create_comment(file, comment)
+    #security_comment = security_reviewer.convert(change)

```

Comment:
I thought maybe this should be merged somehow with the other buddy so kept it in comment for now